examples/bpk-component-input/examples.js

```diff
@@ -98,7 +98,7 @@ const TextExample = () => (
       name="text_value"
       value="Edinburgh"
       onChange={action('input changed')}
-      placeholder="Enter a country, city or airport"
+      placeholder="Enter a country, city or airport..."
```

Comment:
Just checking what the purpose of adding the `...` at the end of this is for?

Reply:
It due to the snapshots cannot be updated at the beginning, so changed the placeholder content here. It seems work now and revise these change.